tentacle/Cargo.toml

```diff
@@ -49,6 +49,8 @@ tokio-rustls = { version = "0.26.0", optional = true }
 # rand 0.8 not support wasm32
 rand = "0.8"
 socket2 = { version = "0.5.0", features = ["all"] }
+shadowsocks = { version = "=1.21.0", default-features = false }
```

Comment:
don‘t use “=x”

Reply:
I looked at the shadowsocks-service library, and I feel that its dependencies are too heavy. We only need a socks5 implementation. Is there a way to switch to a simpler socks5 or write a custom implementation(we just need a handshake process)? Its dependencies are a bunch of unnecessary ones for us.